cmd/argocd/commands/admin/settings_rbac.go

```diff
@@ -228,11 +254,15 @@ a local file, and in either CSV or K8s ConfigMap format.
 					fmt.Printf("Policy is invalid: %v\n", err)
 					os.Exit(1)
 				}
+			} else {
+				log.Fatalf("Policy is empty or could not be loaded.")
 			}
 		},
 	}
 
 	command.Flags().StringVar(&policyFile, "policy-file", "", "path to the policy file to use")
+	command.Flags().StringVar(&policyFile, "namespace", "", "namespace to get argo rbac configmap from")
```

Comment:
the flag `&policyFile` is repeated. This should be `&namespace`?

Reply:
Yup missed this thanks. Addressed